website/client/js/controllers/sortableInventoryCtrl.js

```diff
@@ -1,10 +1,10 @@
 habitrpg.controller('SortableInventoryController', ['$scope',
   function ($scope) {
     var attributeSort = {
-      constitution: ['-con', '-(con+int+per+str)'],
-      intelligence: ['-int', '-(con+int+per+str)'],
-      perception: ['-per', '-(con+int+per+str)'],
-      strength: ['-str', '-(con+int+per+str)'],
+      constitution: ['-(_effectiveCon)', '-(_effectiveCon+_effectiveInt+_effectivePer+_effectiveStr)'],
```

Comment:
This is verbose, but it easily conveys what is happening. I don't think it's worth over engineering the client side code before we do the rewrite anyway.